app/assets/javascripts/assets.js

```diff
@@ -92,11 +50,27 @@ function setupAssetsLoading() {
     });
   }
 
-  intervalId = window.setInterval(function() {
-    cntr += 1;
-    if (cntr >= REPETITIONS || !refreshAssets()) {
-      finalizeAssets();
-      window.clearInterval(intervalId);
-    }
-  }, DELAY);
-}
+  function setupAssetsLoading() {
+    var DELAY = 5000;
+    var REPETITIONS = 60;
+    var cntr = 0;
+    var intervalId;
+
+    if (window.assetsPollingIsRunning) return;
+
+    intervalId = window.setInterval(function() {
```

Comment:
I think here is better use setTimeout recursive function, it will be more simple and use less resources.